### PR TITLE
fix(fiatRates): use contract address to distinguish tokens with same symbol

### LINKED ISSUE #3394: Find exchange rate of ERC20 token by token address, not by token abbreviation
It seems the exchange rate is matched not by the token address, but by the token abbreviation, leading to wrong exchange rate being used.

Good example is Uniswap (UNI) vs Unicorn (UNI) token, where the Unicorn rate is being used when actually Uniswap rate should've been used.

### PR DESCRIPTION
**Pull request by slowbackspace**
fix https://github.com/trezor/trezor-suite/issues/3394

tl;dr:
- old endpoint we were using doesn't provide an option to pass token address, switched to different endpoint just for tokens
- only ethereum supported (coingecko limitation), but I guess that's fine
- now with this new api endpoint we no longer need to fetch list of coins from coingecko (its purpose is to map symbols (even token symbols) to coingecko IDs which were then used in old endpoint)
- small change on DB front, instead of autogenerating key from the symbol I pass key param consisting of symbol and optional token's address. I kept a migration stupid simple. Drop old objectStore and create a new one as there is no point in preserving fiat rates between version as they are updated at least once per 10 mins.

### CODE REVIEW

File: packages/suite/src/services/coingecko.ts [hunk 1]
```diff
@@ -28,41 +29,76 @@ const fetchCoinGecko = async (url: string) => {
...
+
+    const url = `${COINGECKO_API_BASE_URL}/simple/token_price/${
+        networkTickerConfig.coingeckoId
+    }?contract_addresses=${ticker.tokenAddress}&vs_currencies=${FIAT_CONFIG.currencies.join(',')}`;
+
+    const rates = await fetchCoinGecko(url);
+    if (!rates) return null;
+
+    return {
+        ts: new Date().getTime() / 1000,
```
Comment (tomexx) [inline packages/suite/src/services/coingecko.ts:83]: a total nitpick:
We could use a helper fn to create timestamps. I also saw `getUnixTime(new Date()) * 1000` in `suite/src/actions/wallet/fiatRatesActions.ts`

**Review by tomexx — APPROVED** [review]
I like it 👍

File: packages/suite/src/services/coingecko.ts (hunk 1 above)
Comment (slowbackspace) [reply packages/suite/src/services/coingecko.ts:83]: hmmm `getTime` basically already returns unix timestamp, but in milliseconds. Importing another util function and using it versus manually dividing by 1000 seems equal to me. Maybe we could do the change in fiatRatesActions and switch `getUnixTime(new Date()) * 1000` to `new Date().getTime()`? Both should do the same right? I liked date-fns too much it seems 😁 

I use to have problems with such util functions, I never remember if the param should be in milliseconds (what JS does) or in seconds. And because it can't be type-checked it already caused me a headache few times 😁

**Review by szymonlesisz — APPROVED** [review]
looks good. didn't test it

**Comment by szymonlesisz** [comment]
resolve changelog conflicts pls
